apps/pie-docs/src/components/template/template.md

```diff
@@ -2,6 +2,7 @@
 eleventyNavigation:
     key: Template
     parent: Components
-    # url: /components/template/overview/  // should be uncomented for a live component page
+    # url: /components/template/  // should be uncomented for a live component page
```

Comment:
Please could you copy this comment to the other lines that should be uncommented?
```suggestion
    # url: /components/template/  // should be uncommented for a live component page
```